presto-hive/src/test/java/com/facebook/presto/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -371,16 +371,20 @@ public void testScanFilterAndProjectOperator()
 
         assertEquals(driverContext.getSystemMemoryUsage(), 0);
 
-        for (int i = 0; i < 50; i++) {
+        for (int i = 0; i < 54; i++) {
             assertFalse(operator.isFinished());
-            assertNotNull(operator.getOutput());
-            assertBetweenInclusive(driverContext.getSystemMemoryUsage(), 90_000L, 499_999L);
+            operator.getOutput();
+            assertBetweenInclusive(driverContext.getSystemMemoryUsage(), 20_000L, 699_999L);
```

Comment:
Due to the buffering memory consumption has grown. This is the tighter range i could chose.